### ISSUE TITLE
reserved identifier violation

### ISSUE THREAD

**Issue by elfring** [medooze/media-server#33]
I would like to point out that identifiers like “[`_CODECS_H_`](https://github.com/medooze/media-server/blob/371242612e4a4d839301f6cd986cc676c93a955d/include/codecs.h#L1 "Update candidate")” and “[`_MEDIA_H_`](https://github.com/medooze/media-server/blob/f00fd54cb89680165c1008a3ceabb5e0c17401c8/include/media.h#L1 "Another update candidate")” [do not fit](https://www.securecoding.cert.org/confluence/display/cplusplus/DCL51-CPP.+Do+not+declare+or+define+a+reserved+identifier#DCL51-CPP.Donotdeclareordefineareservedidentifier-NoncompliantCodeExample%28HeaderGuard%29 "Do not declare an identifier which is reserved for the compiler implementation.") to the expected naming convention of the C++ language standard.
Would you like to adjust your selection for unique names?

**Comment by murillo128** [comment]
Yes, I recently discovered that defines starting with an underscore are reserved, would you be willing to provide a CR?

**Comment by elfring** [comment]
Would any other software developers like to improve affected places quicker (than me)?
